practice-one/src/constants/error.ts

```diff
@@ -0,0 +1,11 @@
+export const ERROR_MESSAGES = {
```

Comment:
use the messages for file name will inlcude all the message info, success, error...